apps/web/src/contexts/globalLayout/GlobalSidebar/GlobalSidebar.tsx

```diff
@@ -1,17 +1,74 @@
+import { AnimatePresence } from 'framer-motion';
 import { ReactElement } from 'react';
 import styled from 'styled-components';
 
+import breakpoints from '~/components/core/breakpoints';
+import colors from '~/components/core/colors';
+
+import AnimatedSidebarDrawer from './AnimatedSidebarDrawer';
+import { useDrawerState } from './SidebarDrawerContext';
+
+export const GLOBAL_SIDEBAR_DESKTOP_WIDTH = 64;
+export const GLOBAL_SIDEBAR_MOBILE_HEIGHT = 48;
+
 type GlobalSidebarProps = {
   content: ReactElement | null;
 };
 
+const ShowDoneFooter = ['settings'];
+
 export default function GlobalSidebar({ content }: GlobalSidebarProps) {
-  return content ? <StyledSidebar>{content}</StyledSidebar> : null;
+  const drawerState = useDrawerState();
+
+  if (!content) {
+    return null;
+  }
+
+  return (
+    <StyledGlobalSidebar>
+      <StyledGlobalSidebarContent>{content}</StyledGlobalSidebarContent>
+      <AnimatePresence>
```

Comment:
Any shot we can make this feel better? Everything is feeling so nice and this just throws my brain way off

[Screen Recording 2023-03-02 at 10.00.01 AM.mov](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/icIMHMJMNkXxGoH3BEkt/c541fb98-137e-42b7-a6df-bec25fb64aae/Screen%20Recording%202023-03-02%20at%2010.00.01%20AM.mov)

Reply:
I guess ideally they would transition at the same time. Right now it feels like two separate things are happening

Reply:
This is significantly better!!!